lib/netuitive/error_tracker.rb

```diff
@@ -0,0 +1,54 @@
+require 'netuitive/controller_utils'
+require 'netuitive/rails_config_manager'
+module ErrorTrackerHook
+  extend ActiveSupport::Concern
+
+  include ControllerUtils
+
+  def isIgnoredError(exception)
+    if !ConfigManager.ignoredErrors.empty?
+      ConfigManager.ignoredErrors.each do |name|
+        if name.include? "^"
+          name.tr!('^','')
+          exception.class.ancestors.each do |ancestor|
+            if name.casecmp(ancestor.name) == 0
+              return true;
+            end
```

Comment:
you could just go with `return true if name.casecmp(ancestor.name) == 0`. No need to have the if/end